packages/db/src/abstractRepository.ts

```diff
@@ -63,14 +78,18 @@ export abstract class Repository<I extends Id, T> {
   }
 
   async put(id: I, value: T): Promise<void> {
+    this.dbWriteMetrics?.inc();
     await this.db.put(this.encodeKey(id), this.encodeValue(value));
   }
 
   async putBinary(id: I, value: Buffer): Promise<void> {
+    this.dbWriteMetrics?.inc();
     await this.db.put(this.encodeKey(id), value);
   }
 
   async delete(id: I): Promise<void> {
+    // do we wanna intorduce db deletes or treat delete as write?
+    // this.dbWriteMetrics?.inc();
```

Comment:
I think threating delete as write is a great idea, let's do it

Reply:
Does it make more sense to add delete metric instead?

Reply:
I don't think so, I would just do `this.dbWriteMetrics?.inc();`